rust/bin-shared/src/tun_device_manager/linux.rs

```diff
@@ -245,35 +237,28 @@ fn make_route_v6(idx: u32, handle: &Handle, route: Ipv6Network) -> RouteAddReque
         .destination_prefix(route.network_address(), route.netmask())
 }
 
-async fn add_route(route: &IpNetwork, idx: u32, handle: &Handle) -> Result<()> {
+async fn add_route(route: &IpNetwork, idx: u32, handle: &Handle) {
     let res = match route {
         IpNetwork::V4(ipnet) => make_route_v4(idx, handle, *ipnet).execute().await,
         IpNetwork::V6(ipnet) => make_route_v6(idx, handle, *ipnet).execute().await,
     };
 
-    match res {
-        Ok(_) => {}
-        Err(NetlinkError(err)) if err.raw_code() == FILE_ALREADY_EXISTS => {}
-        // TODO: we should be able to surface this error and handle it depending on
-        // if any of the added routes succeeded.
-        Err(err) => Err(err).context("Failed to add route")?,
+    if let Err(err) = res {
+        tracing::debug!(%route, "Failed to add route: {err}");
```

Comment:
Maybe we can keep the check for existing routes to not spam the log here? Being idempotent is not an error :)